libs/domain/picking/src/components/picking-list-item/picking-list-item.component.ts

```diff
@@ -0,0 +1,129 @@
+import { resolve } from '@spryker-oryx/di';
+import { PickingListService } from '@spryker-oryx/picking';
+import { IconTypes } from '@spryker-oryx/themes/icons';
+import { Size } from '@spryker-oryx/ui';
+import { ButtonType } from '@spryker-oryx/ui/button';
+import {
+  asyncState,
+  i18n,
+  isDefined,
+  observe,
+  valueType,
+} from '@spryker-oryx/utilities';
+import { html, LitElement, TemplateResult } from 'lit';
+import { property, state } from 'lit/decorators.js';
+import { when } from 'lit/directives/when.js';
+import { BehaviorSubject, distinctUntilChanged, filter, switchMap } from 'rxjs';
+import { styles } from './picking-list-item.styles';
+
+export class PickingListItemComponent extends LitElement {
+  static styles = styles;
+
+  protected pickingListService = resolve(PickingListService);
+
+  @property() pickingListId?: string;
+
+  @observe()
+  protected pickingListId$ = new BehaviorSubject(this.pickingListId);
+
+  @state()
+  protected isDisabled?: boolean;
```

Comment:
Leave a comment please, that this should be refactored when implement start picking logic and loading state should be area of responsibility of the service.